### PR TITLE
Set up the new profile design

### PR DESCRIPTION
**Pull request by Smartman01**
### Summary

Set up the new profile design with the fragments, and change the homepage layout. Also add gallery feature to pick image and saves to sharedPreferences. 
### Screenshots/GIFs

Screenshot_1580502101

Screenshot_1580537800
Screenshot_1580537871
Screenshot_1580537864

### CODE REVIEW

File: app/src/main/res/layout/activity_profile.xml [hunk 1]
```diff
@@ -10,78 +10,51 @@
...
-        android:layout_alignParentEnd="true"
-        android:layout_alignParentRight="true"
-        android:layout_centerVertical="true"
-        android:layout_centerHorizontal="true"
-        android:layout_centerInParent="true"
-        android:layout_marginTop="28dp"
-        android:layout_marginEnd="81dp"
-        android:layout_marginRight="81dp"
-        app:srcCompat="@drawable/ic_qr_code" />
+        <!--fragment
```
Comment (krabinowitz1) [inline app/src/main/res/layout/activity_profile.xml]: if you don't need this commented out code, could you please remove it from the file (:

File: app/src/main/res/layout/activity_profile.xml [hunk 2]
```diff
@@ -10,78 +10,51 @@
...
         android:textSize="40sp"
-        android:textAlignment="center"
         android:textStyle="bold"
         android:layout_centerHorizontal="true"
         tools:layout_editor_absoluteX="0dp"
         tools:layout_editor_absoluteY="0dp" />
 
-    <TextView
-        android:id="@+id/scan_code_text_view"
+    <FrameLayout
```
Comment (krabinowitz1) [inline app/src/main/res/layout/activity_profile.xml]: You're going to need to give this an ID.

File: app/src/main/res/layout/fragment_email_login.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (krabinowitz1) [inline app/src/main/res/layout/fragment_email_login.xml]: The FrameLayout is only necessary inside the activity layout, not the fragment layouts so if you could remove them from each of the fragment xml files that would be great!

**Comment by krabinowitz1** [comment]
@Smartman01 do you know why the cardview shadows look funky in some of the ui images? Also do you think you can try to make the view with the points look more like how it does on notion? It's supposed to have rounded corners and it needs some padding around the text. The same goes for the "Enter email code" edit text. It looks a little plain and could use some styling like how it looks on notion.

**Comment by Smartman01** [comment]
No need to worry about the shadows, those appear only in android studio not on an actual device.

**Pull request merged**